lib/action_controller/serialization.rb

```diff
@@ -57,5 +61,13 @@ def serialization_scope(scope)
         self._serialization_scope = scope
       end
     end
+
+    def original_url
+      request.original_url.sub(/\?.*$/, "")
```

Comment:
@bf4 I don't what I am doing wrong but it seems the exactly the same.

```
(byebug) "http://localhost:3000/author?teste=teste#L21"[/\A[^?]+/]
"http://localhost:3000/author"
(byebug) "http://localhost:3000/author?teste=teste#L21".sub(/\?.*$/, "")
"http://localhost:3000/author"
```

Was it your idea?